native/src/modules/app/constants/NavigationTypes.js

```diff
@@ -24,10 +24,11 @@ import type {
   WohnenOfferRouteType
 } from 'api-client/src/routes'
 import type { FeedbackInformationType } from '../../../routes/feedback/containers/FeedbackModalContainer'
-import type { RedirectRouteType } from 'api-client'
+import type { JpalEvaluationRouteType, RedirectRouteType } from 'api-client'
```

Comment:
Think it would be nicer to import all routes from 'api-client' not some from 'api-client/src/routes' or is this for a reason?